docs/section-2-using-parsers.md

```diff
@@ -615,6 +615,28 @@ For example, this pattern would match any node inside a call:
 (call (_) @call.inner)
 ```
 
+#### Special Nodes
+
+When the parser encounters text it does not recognize, it represents this node
+as `(ERROR)` in the syntax tree. These error nodes can be queried just like
+normal nodes:
+
+```scheme
+(ERROR) @the-error
+```
+
+Similarly, if a parser is almost able to parse a document but it detects a
+missing node, it will insert that missing node into the tree to preserve the
+rest of the document. These missing nodes appear as regular nodes in the tree
+(but with zero width), and can be queried using the `(MISSING)` alias:
+
+```scheme
+(MISSING) @missing-node
+```
+
+This is useful when trying to detect all syntax errors, since missing nodes will
+not be covered by `(ERROR)` queries.
```

Comment:
```suggestion
This is useful when attempting to detect all syntax errors in a given parse tree, since these missing node are not captured by `(ERROR)` queries.
```